packages/sdk/src/swap/__tests__/sdk.test.ts

```diff
@@ -728,7 +728,7 @@ describe(SwapSDK, () => {
         amount: BigInt(1e18).toString(),
         fillOrKillParams: {
           retryDurationBlocks: 500,
-          refundAddress: '0xa56A6be23b6Cf39D9448FF6e897C29c41c8fbDFF',
+          refundAddress: 'tb1qw508d6qejxtdg4y5r3zarvary0c5xw7kxpjzsx',
```

Comment:
changed because validation now checks if refund address is valid for source chain